src/scicookie/{{cookiecutter.project_slug}}/docs-mkdocs/mkdocs.yaml

```diff
@@ -1,10 +1,18 @@
 site_name: {{ cookiecutter.project_name}}
 site_url: {{ cookiecutter.project_url}}
 repo_url: {{ cookiecutter.git_https_upstream}}
-docs_dir: ./
-site_dir: ../build
+docs_dir: ./docs/
```

Comment:
this is what I have for makim: https://github.com/osl-incubator/makim/blob/main/docs/mkdocs.yaml#L4-L5